### PR TITLE
Updated acceptance tests for staging

### PR DESCRIPTION
**Pull request by tomrosier**
Theses tests should check if the proxy node is running.

### CODE REVIEW

File: proxy-node-acceptance-tests/features/proxy_node.feature [hunk 1]
```diff
@@ -2,22 +2,29 @@ Feature: proxy-node feature
 
     Scenario: Proxy node happy path
         Given the user is at Stub Connector
-        And they login as "stub-idp-demo"
+        And they progress through verify
+        And they login to stub idp
         Then they should arrive at the success page
 
     Scenario: IDP Generates Authn Failure
```
Comment (JHjava) [inline proxy-node-acceptance-tests/features/proxy_node.feature]: Do we need to test this?

Comment (tomrosier) [reply proxy-node-acceptance-tests/features/proxy_node.feature]: Come to think of it probably not.

File: proxy-node-acceptance-tests/run-staging-tests.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env sh
+set -u
+
+echo "Before Docker compose build"
+docker-compose build
+echo "Docker compose build"
+export PROXY_NODE_URL="https://test-connector.staging.verify.govsvc.uk"
+export STUB_CONNECTOR_URL="https://test-connector.staging.verify.govsvc.uk"
+export STUB_IDP_USER="stub-idp-demo-one"
+docker-compose up --abort-on-container-exit | grep acceptance-tests_1 --colour=never
```
Comment (vixus0) [inline proxy-node-acceptance-tests/run-staging-tests.sh]:
```suggestion
docker-compose up --abort-on-container-exit --detach
docker-compose logs acceptance-tests --no-color
```

File: proxy-node-acceptance-tests/run-staging-tests.sh [hunk 3]
```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env sh
+set -u
```
Comment (vixus0) [inline proxy-node-acceptance-tests/run-staging-tests.sh]:
```suggestion
set -eu
```

File: proxy-node-acceptance-tests/run-tests-with-docker.sh [hunk 4]
```diff
@@ -4,7 +4,9 @@ set -u
 echo "Before Docker compose build"
 docker-compose build
 echo "Docker compose build"
-export PROXY_NODE_URL="http://$(minikube ip)"
+export PROXY_NODE_URL="http://$(minikube ip):31200"
+export STUB_CONNECTOR_URL="http://$(minikube ip):31200"
```
Comment (vixus0) [inline proxy-node-acceptance-tests/run-tests-with-docker.sh]:
```suggestion
export STUB_CONNECTOR_URL="http://$(minikube ip):31100"
```

File: stub-connector/src/main/java/uk/gov/ida/notification/stubconnector/resources/SendAuthnRequestResource.java [hunk 5]
```diff
@@ -68,6 +69,11 @@ public Response invalidAuthnRequest(
         @Context HttpServletResponse httpServletResponse
     ) throws ResolverException, ComponentInitializationException, MessageHandlerException, MessageEncodingException {
         MessageContext context = generateAuthnRequestContext(session);
+
+        AuthnRequest authenRequest = (AuthnRequest) context.getMessage();
```
Comment (vixus0) [inline stub-connector/src/main/java/uk/gov/ida/notification/stubconnector/resources/SendAuthnRequestResource.java]: `s/authenRequest/authnRequest/g`

File: proxy-node-acceptance-tests/run-staging-tests.sh (hunk 3 above)
Comment (tomrosier) [reply proxy-node-acceptance-tests/run-staging-tests.sh]: 🇪🇺

**Pull request merged**